mpop_service/logutils.cpp

```diff
@@ -0,0 +1,91 @@
+#include "logutils.h"
+#include <QTime>
+#include <QFile>
+#include <QFileInfo>
+#include <QDebug>
+#include <QDir>
+#include <QFileInfoList>
+#include <iostream>
+
+static QString logFileName;
+static QString logFolderPath;
+
+logutils::logutils()
+{
+    // set logfiles path.
+    logFolderPath = "/var/log/";
+
+    deleteOldLogs(); //delete old log files
```

Comment:
So, this is executed each time the service is started? All previous log files are deleted?